src/cocotb/handle.py

```diff
@@ -1261,6 +1261,30 @@ def __str__(self) -> str:
         return self.value.decode("ascii")
 
 
+class PackedStructObject(LogicObject, HierarchyObject):
+    """
+    Incorperates the functionality of both HierarchyObject and LogicObject
+
+    For getting struct children with a name collision, use handle["name"]
+
+    Since this class lists LogicObject first for inheritance, conflicting methods will call LogicObject's method
+    """
+
+    def __init__(self, handle, path):
+        # Use HierarchyObject because _sub_handles contains SimHandleBase, which is a broader type
+        HierarchyObject.__init__(self, handle, path)
+
+    def __setattr__(self, name: str, value: Any) -> None:
+        if name == "value":
+            return LogicObject.__setattr__(self, "value", value)
+
+        return HierarchyObject.__setattr__(self, name, value)
```

Comment:
I don't think this is necessary? Attempting to set `.value` should always trigger the setter property since `LogicObject` comes before `HierarchyObject` in the inheritance hierarchy.